website/server/libs/amazonPayments.js

```
@@ -196,14 +196,19 @@ api.cancelSubscription = async function cancelSubscription (options = {}) {
 
   let details = await this.getBillingAgreementDetails({
     AmazonBillingAgreementId: billingAgreementId,
+  }).catch(function errorCatch (err) {
+    return err;
```

same here, the error gets ignored

---

Do we have a log library?

---

yes, libs/logger and it uses winston